sqlite/comdb2build.c

```diff
@@ -1913,9 +1921,23 @@ static void free_ddl_context(Parse *pParse)
 static int comdb2_parse_sql_type(const char *type, int *size)
 {
     char *endptr;
+    size_t type_len;
+    int size_maybe;
+
+    type_len = strlen(type);
 
     for (int i = 0;
          i < sizeof(type_mapping) / sizeof(struct comdb2_type_mapping); ++i) {
+
+        /* Check if current type could accept size. */
+        int size_maybe =
```

Comment:
Also the name "_maybe_" sounds confusing. Change it to something like "_accepts_size_"?